common/chains/client/multi_node_client.go

```diff
@@ -0,0 +1,310 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+
+	"github.com/smartcontractkit/chainlink/v2/common/types"
+)
+
+var (
+	// PromMultiNodeClientRPCNodeStates reports current RPC node state
+	PromMultiNodeClientRPCNodeStates = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "client_rpc_node_states",
+		Help: "The number of RPC nodes currently in the given state for the given chain",
+	}, []string{"chainFamily", "chainId", "state"})
```

Comment:
Oh I think there is actually precedent for calling this `network` instead:
```suggestion
	}, []string{"network", "chainId", "state"})
```

Reply:
fixed